src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/annotation/missingoverride/Example1.java

```diff
@@ -0,0 +1,42 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="MissingOverride"/>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.annotation.missingoverride;
+
+// xdoc section -- start
```

Comment:
this comment define a border of what user see, lets move this comment down to avoid this class being visible in xdoc files.